### ISSUE TITLE
Option to more heavily customise Agama workflow

### ISSUE THREAD

**Issue by sysrich** [agama-project/agama#601]
I'd really like to use Agama for openSUSE Aeon, but one of the core goals of Aeon is to be much more prescriptive in its delivery. 

I want all my users to pretty much get the same thing, with the same options, with very limited scope for altering that.

For example, the current way Agama handles Software seems to be PERFECT for that, +100, awesome. I don't want that to change.

However, I'd also like an ability to do the following (these can be split out to other issues, but given the user story I bundle them all here to start with)

- Hide/Remove the option to customise Localization - Aeon can handle this on firstboot. 
- Hide/remove the option for a regular user. Force the user to only configure root authentication - Aeon can also handle regular user creation on firstboot, and I need that workflow to give the user a chance to customise their desktop options while setting up the user account.
- Hide/remove the option for Storage > LVM - I don't want to support LVM on Aeon
- Have some way of toggling Storage > Use encryption to be on by default - I want Aeon to use the awesome new FDE stuff as much as possible. This will of course need to also force users to enter a passphrase by default.

Some of this may be possible already, but I haven't gotten my head around all the options in agama.yaml just yet, hence this issue to explain where I'm coming from

Thanks!

**Comment by sfalken** [comment]
For my part, with Kalpa, at present, I'm going to need the user creation functionality, but making it something that you can toggle on/off as needed is definitely something I can get behind.

**Comment by sysrich** [comment]
Yeah that's why I'm pushing for #602 ;) if Kalpa and Aeon are both gonna use Agama we're gonna need different options for quite a few settings that already exist, as well as some of the new ones I'm asking for here :)

**Comment by imobachgs** [comment]
I am closing this bug report because neither Kalpa nor Aeon will use Agama. We have plans to offer a way to extend Agama, but it will take some time (we have other priorities).

Thanks!
